### PR TITLE
Implemented the B.F.S In PHP

### LINKED ISSUE #2171: Breadth First Search
- [x] C
- [x] C++
- [x] C# - @prashansag62
- [ ] CoffeeScript
- [x] Dart - @Rukmini-Meda
- [ ] Go - @Aju100
- [x] Java
- [x] JS
- [ ] Kotlin
- [ ] Php - @srimani-programmer
- [x] Python
- [x] Readme
- [x] Ruby - @HardevKhandhar
- [ ] Rust
- [ ] TypeScript

### PR DESCRIPTION
**Pull request by srimani-programmer**
Fixes #2171 

#### Checklist:
- [x] 4 space indentation.
- [x] Coding conventions are followed.
- [x] Input is taken dynamically.
- [x] Sample Input / Output is added at the end of file.
- [x] Logic Documentation (Comments).
- [x] File names are correct.

Make sure these boxes are checked before your pull request (PR) is ready to be reviewed and merged. Thanks!

#### Changes proposed in this pull request:
- Implemented the Breadth-First Search (B.F.S) in PHP

#### Languages Used:
- php

#### Files Added:
- Breadth_First_Search.php

> We're happy to help you get this ready -- don't be afraid to ask for help.

Thanks!

### CODE REVIEW

File: Breadth_First_Search/Breadth_First_Search.php [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+// output:
+// least number of A to C in 1 hops
+// A->C
+
+// least number of hops between A and G
+$graphObject->breadthFirstSearch('A', 'F');
+// output:
+// least number of hops between A and F
+// A->B->D->->F
+?>
```
Comment (singh-shreya6) [inline Breadth_First_Search/Breadth_First_Search.php:108]: Please add a blank line at end of file.

File: Breadth_First_Search/Breadth_First_Search.php [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+    'D' => array('A', 'F'),
+    'E' => array('B', 'D', 'F', 'A', 'C'),
+    'F' => array('A', 'E', 'C'),
+  );
+
+$graphObject = new Graph($graph);
+
+//Least number of hops between B and F
+$graphObject->breadthFirstSearch('B', 'F');
+// output:
```
Comment (singh-shreya6) [inline Breadth_First_Search/Breadth_First_Search.php:87]: Please keep code here. You can add all Sample Inputs and Sample Outputs as multi line comment at end of file.

File: Breadth_First_Search/Breadth_First_Search.php [hunk 3]
```diff
@@ -0,0 +1,108 @@
...
+  protected $visitedNode = array();
+
+  public function __construct($graph) {
+    $this->graph = $graph;
+  }
+
+  // find least number of hops (edges) between 2 nodes
+  // (vertices)
+  public function breadthFirstSearch($origin, $destination) {
+    // mark all nodes as unvisited
```
Comment (singh-shreya6) [inline Breadth_First_Search/Breadth_First_Search.php:14]: Please ensure 4 space indentation after function start, if or for start.

File: Breadth_First_Search/Breadth_First_Search.php [hunk 4]
```diff
@@ -0,0 +1,108 @@
+<?php
```
Comment (singh-shreya6) [inline Breadth_First_Search/Breadth_First_Search.php:1]: Add a brief problem description on top. 1-2 lines.

**Comment by somya-kapoor** [comment]
Are you working on it?

**Comment by singh-shreya6** [comment]
No changes from 12 days. Closing this PR. Please raise a new one with the suggested changes if you want to continue on it.

**Pull request closed without merging**